content/cloud-docs/registry/add.mdx

```diff
@@ -17,8 +17,6 @@ You can add providers and modules from the public [Terraform Registry](/registry
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
```

Comment:
My suggestion would be to update the first paragraph as follows:

"You can add providers and modules from the public [Terraform Registry](/registry) to your Terraform Cloud private registry. The private registry stores a pointer to these public providers and modules so that you can view their data from within Terraform Cloud. This lets you clearly designate which public providers and modules are recommended for the organization and makes their supporting documentation and examples centrally accessible."

Rather than removing the offending paragraph entirely, I think we should do something like what I have above because it helps explain how it actually works. I think if we remove it entirely we could get folks wondering if we are caching, or mirroring, or whatever.

Reply:
Awesome - that makes sense to me. I'll make the change you are suggesting.

Reply:
I just updated the wording based on your feedback.